lib/cloud/gcp/vm.go

```diff
@@ -69,6 +76,15 @@ func convertAPIError(err error) error {
 			return trail.FromGRPC(apiError)
 		}
 	}
+	var urlError *url.Error
+	if errors.As(err, &urlError) {
+		if codeStr := urlErrorCodePattern.FindString(urlError.Error()); codeStr != "" {
```

Comment:
This seems brittle. Do you even need it anymore? If all you're doing now is logging the error, who cares if it's a trace error or not?